theano/gpuarray/extra_ops.py

```diff
@@ -450,12 +450,13 @@ def c_support_code_struct(self, node, nodename):
         """ % locals()
         return super(GpuCumsum, self).c_support_code_struct(node, nodename) + code
 
-
+@register_opt('fast_compile')
 @op_lifter([CumsumOp])
-def use_gpu_cumsumop(node, ctx_name):
-    if node.inputs[0].dtype == 'float32':
-        axis = node.op.axis
-        x = node.inputs[0]
+@register_opt2([CumsumOp], 'fast_compile')
```

Comment:
(Sorry, I had pasted the wrong command in my previous comment.)

I had just tested the latter comment. Code remains the way it is in the first comment. For reproducing,
`THEANO_FLAGS=device=cuda,lib.cnmem=0.6 theano-nose theano/gpuarray/tests/test_extra_ops.py`